src/debugtopus.ts

```diff
@@ -101,6 +102,25 @@ export const runTest = async ({
   packageRootDir: string;
   runMode: "ui" | "headless";
 }): Promise<void> => {
+  const file = chromium.executablePath();
+
+  // todo: need to await for errored promise
+  await access(file, fs.constants.X_OK, async (error) => {
+    if (error && error.code === "ENOENT") {
+      console.log(
+        "Couldn't find chromium executable, some message about running 'npx playwright install'..."
+      );
+
+      const someVariable = promisify(exec)("npx playwright install", {
+        cwd: packageRootDir,
+      });
+
+      someVariable.child.stdout!.on("data", (data) => console.log(data));
+
+      await someVariable; // todo:  this does not work
+    }
+  });
+
```

Comment:
I had it working as expected like this

```typescript
  const file = chromium.executablePath();
  try {
    await fs.access(file, fs.constants.X_OK, );
  } catch (e) {
    const error = e as Record<string, string>;
    if ("code" in error && error.code === "ENOENT") {
      console.log(
        "Couldn't find chromium executable, some message about running 'npx playwright install'..."
      );
      const someVariable = promisify(exec)("npx playwright install", {
        cwd: packageRootDir,
      });
      someVariable.child.stdout!.on("data", (data) => console.log(data));

      await someVariable;
    }
  }
```

but I like commited code more, I just don't know how to await "error promise"